src/vaadin-date-time-picker.html

```diff
@@ -19,25 +22,68 @@
         display: none !important;
       }
     </style>
-    <slot></slot>
+    <vaadin-custom-field id="customField" on-value-changed="__customFieldValueChanged" theme$="[[theme]]">
+      <slot name="date-picker">
+        <vaadin-date-picker part="date" theme$="[[theme]]"></vaadin-date-picker>
+      </slot>
+      <slot name="time-picker">
+        <vaadin-time-picker part="time" theme$="[[theme]]"></vaadin-time-picker>
+      </slot>
+    </vaadin-custom-field>
   </template>
 
   <script>
     (function() {
 
+      /* eslint-disable max-len */
```

Comment:
The problem is only the long link URLs. This is the longest:
```
*   [`<vaadin-custom-field>` documentation](https://vaadin.com/components/vaadin-custom-field/html-api/elements/Vaadin.CustomFieldElement)
```
When a line only contains a single link and is still "too long" I think this `eslint-disable max-len` is necessary. I wouldn't like to break consistency with link text here even if it would barely be under the linter max length limit. Since we can't directly control how long the URL is at vaadin.com even if we started shortening the link texts we still might run into this issue and would need to use `eslint-disable max-len`.

Reply:
Yeah, but I was thinking that we can use shorter names for all components in that documentation list, like: `['custom-field' documentation]`, `['date-picker' documentation]`, `['time-picker' documentation]`. 

I have checked it previously, it will pass even with that long URL.
It's a nit, just want to avoid skipping linter even in the docs.

Reply:
Links shortened and removed `eslint-disable max-len`.